src/libraries/System.Text.Json/src/System/Text/Json/ThrowHelper.Serialization.cs

```diff
@@ -19,7 +19,6 @@ public static void ThrowArgumentException_DeserializeWrongType(Type type, object
             throw new ArgumentException(SR.Format(SR.DeserializeWrongType, type, value.GetType()));
         }
 
-        [DoesNotReturn]
```

Comment:
I am curious, how did you catch this typo?

Reply:
A new version of the compiler checks method bodies to see that they honor the nullability attributes that they declare in their API.  This method has `return` statements, so it was flagged ;-)

Reply:
Nice!